chainstate/storage/src/schema.rs

```diff
@@ -56,16 +54,20 @@ storage::decl_schema! {
         /// Store the number of transactions per account
         pub DBAccountNonceCount: Map<AccountType, AccountNonce>,
 
+        pub DBTokensData: Map<TokenId, tokens_accounting::TokenData>,
```

Comment:
I think I missed bumping `ChainstateStorageVersion` after adding new fields

Reply:
Yes, please do that.